src/util/system.cpp

```diff
@@ -359,6 +374,19 @@ Optional<unsigned int> ArgsManager::GetArgFlags(const std::string& name) const
     return nullopt;
 }
 
+bool ArgsManager::GetCommand(std::string& cmd, std::vector<std::string>& args) const
+{
+    LOCK(cs_args);
+    assert(!m_accept_any_command); // AddCmd must have been called before this
```

Comment:
I wonder if it wouldn't be better to always copy the remaining args into `m_command` and have the only difference be that you get an "unknown command" when `m_accept_any_command` is set

Reply:
Thanks. Fixed, I think